src/template.html

```diff
@@ -6,6 +6,13 @@
     <title>Document</title>
 </head>
 <body>
-    <h1>hello world</h1>
+    <header>
+        <div id="logo"><h3>PokeFiles</h3></div>
```

Comment:
- [ ] OPTIONAL SUGGESTION - It is generally recommended to include an ` <h1> `tag for better accessibility and semantic structure describing the main content of the page in this case Pokefiles. Maybe you can consider changing that 😃

Reply:
Thank you for pointing that out. My reasoning for going with the `<h3>` tag in this case was because `Pokefiles` was meant to be a placeholder for the logo I was going to come up with later, and I thought that using a `<h1>`tag would be too big in the header. However, since the text wont make it to the final build I don't mind changing it in the next PR 🙂